### PR TITLE
feat(savebar): shows form errors and save/cancel buttons

### PR DESCRIPTION
**Pull request by saithis**
To improve tree shaking the ps-form-field and ps-form-errors were moved to their own modules

### CODE REVIEW

File: projects/components/form-base/src/form-base.module.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { PsFormService } from './form.service';
+
+@NgModule({
+  imports: [],
+  declarations: [],
+  exports: [],
+})
```
Comment (MrBreZe) [inline projects/components/form-base/src/form-base.module.ts]: imports: [],
  declarations: [],
  exports: [], könnte weg

Comment (saithis) [reply projects/components/form-base/src/form-base.module.ts]: done

File: projects/components/savebar/src/savebar.component.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  bottom: -1px;
+  margin: 1em 0 1em 0;
+  z-index: 5;
+  padding: 1em !important;
+  display: flex;
+  justify-content: space-between;
+  border-top: solid var(--ps-accent) 5px;
+}
+
+mat-card.ps-savebar__sticky_mode {
```
[the remark this answers is not in the document]
Comment (saithis) [reply projects/components/savebar/src/savebar.component.scss]: done

[the remark this answers is not in the document]
Comment (saithis) [reply projects/components/savebar/src/savebar.component.scss]: done

File: projects/components/savebar/src/savebar.component.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    margin: 0.25em;
+  }
+}
+
+.ps-savebar__container-spacer {
+  margin-right: auto;
+}
+
+@media only screen and (max-width: 35em) {
+  mat-card.ps-savebar__sticky_mode {
```
[the remark this answers is not in the document]
Comment (saithis) [reply projects/components/savebar/src/savebar.component.scss]: done

**Pull request merged**